### ISSUE TITLE
Is dialogflow cx phone gateway can be enabled for taking input from user's phone?

### ISSUE THREAD

**Issue by arshikasoni** [sadasystems/covid19-healthcare-chatbot#2]
Hi,
I want to know that whether we can integrate this bot to Dialogflow CX Phone Gateway and then we can make something like press 1 for this and press 2 for this? and then that bot should work on numbers entered by user?
Actually i have done a research regarding this and found that we have dtmf option which let us take input from the user but that is not working , so can you please let me know if something like this is supported?

**Comment by Yash-Kavaiya** [comment]
Yes, you can integrate a bot with Dialogflow CX Phone Gateway and use DTMF to collect input from the user.
@arshikasoni fixes #2
1. Create a Dialogflow CX agent.
2. Enable DTMF for your agent.
3. Create a page in your agent to handle DTMF input.
4. Add a condition to the page to check the value of the DTMF input parameter.
5. Add a transition to the page for each possible DTMF input value.
![image](https://github.com/sadasystems/covid19-healthcare-chatbot/assets/103239923/329c2b3d-50e9-4dcc-94d6-69a68f947e4c)
